### PR TITLE
Add power command to System Bridge docs

### PR DESCRIPTION
**Pull request by timmo001**
## Proposed change

For https://github.com/home-assistant/core/pull/95719

## Type of change

- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/95719

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/system_bridge.markdown [hunk 1]
```diff
@@ -137,3 +137,23 @@ data:
   bridge: "deviceid"
   text: "Hello"
 ```
+
+### Service `system_bridge.power_command`
+
+Sends power command to system.
+
```
Comment (joostlek) [inline source/_integrations/system_bridge.markdown:147]: Please add the my link for this service

**Review by joostlek — DISMISSED** [review]

File: source/_integrations/system_bridge.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/system_bridge.markdown:147]: We don't do that everywhere yet. 

edit: Ah, it is about consistency on this page. Added it 👍

File: source/_integrations/system_bridge.markdown [hunk 2]
```diff
@@ -137,3 +137,23 @@ data:
   bridge: "deviceid"
   text: "Hello"
 ```
+
+### Service `system_bridge.power_command`
+
+Sends power command to system.
```
Comment (frenck) [inline source/_integrations/system_bridge.markdown]:
```suggestion
Sends power command to the system.

{% my developer_call_service service="system_bridge.power_command" title="Show service in your Home Assistant instance." %}

```

**Review by frenck — APPROVED** [review]
Thanks, @timmo001 👍 

../Frenck

**Pull request merged**